pkgs/sdk_triage_bot/bin/triage.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:args/args.dart';
+import 'package:github/github.dart';
+import 'package:http/http.dart' as http;
+import 'package:sdk_triage_bot/src/common.dart';
+import 'package:sdk_triage_bot/src/gemini.dart';
+import 'package:sdk_triage_bot/src/github.dart';
+import 'package:sdk_triage_bot/triage.dart';
+
+void main(List<String> arguments) async {
+  final argParser = ArgParser();
+  argParser.addFlag('dry-run',
+      negatable: false,
+      help: 'Perform triage but don\'t make any actual changes to the issue.');
+  argParser.addFlag('force',
+      negatable: false,
+      help: 'Make changes to the issue even if it already looks triaged.');
+  argParser.addFlag('help',
+      abbr: 'h', negatable: false, help: 'Print this usage information.');
+
+  final ArgResults results;
+  try {
+    results = argParser.parse(arguments);
+  } on ArgParserException catch (e) {
+    print(e.message);
+    print('');
+    print(usage);
+    print('');
+    print(argParser.usage);
+    io.exit(1);
```

Comment:
[optional] There is no real standard, but I've seen 64 used as the exit code for incorrect usage (I use it as a signal to omit some failing commands from CLI history)

```suggestion
    io.exit(64);
```